netdata-installer.sh

```diff
@@ -628,6 +634,80 @@ bundle_libwebsockets
 
 # -----------------------------------------------------------------------------
 
+build_judy() {
+  local env_cmd=''
+
+  if [ -z "${DONT_SCRUB_CFLAGS_EVEN_THOUGH_IT_MAY_BREAK_THINGS}" ]; then
+    env_cmd="env CFLAGS= CXXFLAGS= LDFLAGS="
+  fi
+
+  pushd "${1}" > /dev/null || exit 1
+  run ${env_cmd} libtoolize --force --copy
+  run ${env_cmd} aclocal
+  run ${env_cmd} autoheader
+  run ${env_cmd} automake --add-missing --force --copy --include-deps
+  run ${env_cmd} autoconf
+  run ${env_cmd} ./configure
+  run ${env_cmd} make -C src
+  run ${env_cmd} ar -r src/libJudy.a src/Judy*/*.o
+  popd > /dev/null || exit 1
+}
+
+copy_judy() {
+  target_dir="${PWD}/externaldeps/libjudy"
+
+  run mkdir -p "${target_dir}" || return 1
+
+  run cp "${1}/src/libJudy.a" "${target_dir}/libJudy.a" || return 1
+  run cp "${1}/src/Judy.h" "${target_dir}/Judy.h" || return 1
+}
+
+bundle_judy() {
+  # If --build-judy flag or no Judy on the system and we're building the dbengine, bundle our own libJudy.
+  if [ -n "${NETDATA_DISABLE_DBENGINE}" ] && [ -z "${NETDATA_BUILD_JUDY}" ] && [ -e /usr/include/Judy.h ] ; then
```

Comment:
would be nice to get some more info maybe. e.g. `/usr/include/Judy.h does not exist. Will not use system Judy but bundle our own`?

Reply:
Good idea, I'll add this.